package.json

```diff
@@ -26,7 +26,7 @@
     "release-minor": "aegir release --type minor --no-parallel",
     "release-major": "aegir release --type major --no-parallel",
     "test": "aegir test --no-parallel --timeout 50000",
-    "test:node": "aegir test --target node --no-parallel",
+    "test:node": "aegir test --target node --no-parallel --timeout 50000",
```

Comment:
I really prefer to see these timeouts in tests. It will help us target batch by batch and do improvs across the board :)

Reply:
feel free, but I don't think it is that useful compared to the amount of time it costs to increase timeouts in each tests manually

Reply:
(and how often that is breaking)